### PR TITLE
Add example of a contract that gives and revokes use objects based on escrowed assets

### PR DESCRIPTION
**Pull request by katelynsills**
This PR adds a use object example contract. The contract gives a use object for coloring pixels, given that the pixels have been escrowed. The user can get their escrowed assets back by calling `completeObj.complete()` but if they do so, the use object functions error with an understandable error message.

More work could be done in the future to make this generic and move the contract into zoe/src/contracts/.

### CODE REVIEW

**Review by erights — APPROVED** [review]
LGTM. Nice!

**Review by erights — APPROVED** [review]

**Comment by katelynsills** [comment]
Great! I added some more comments, let me know if you want another review before merging

**Review by erights — APPROVED** [review]
Still LGTM

**Pull request merged**
